doc/classes/Image.xml

```diff
@@ -166,52 +175,56 @@
 			<argument index="0" name="color" type="Color">
 			</argument>
 			<description>
-				Fills an [code]Image[/code] with a specified [Color]
+				Fills this [code]Image[/code] with a specified [Color].
 			</description>
 		</method>
 		<method name="fix_alpha_edges">
 			<return type="void">
 			</return>
 			<description>
+				Blends the pixels with low alpha values with colors of nearby pixels.
 			</description>
 		</method>
 		<method name="flip_x">
 			<return type="void">
 			</return>
 			<description>
+				Flips the image horizontally.
 			</description>
 		</method>
 		<method name="flip_y">
 			<return type="void">
 			</return>
 			<description>
+				Flips the image vertically.
 			</description>
 		</method>
 		<method name="generate_mipmaps">
 			<return type="int" enum="Error">
 			</return>
 			<description>
+				Generates mipmaps for the image.
 			</description>
 		</method>
 		<method name="get_data" qualifiers="const">
 			<return type="PoolByteArray">
 			</return>
 			<description>
-				Return the raw data of the [code]Image[/code].
+				Returns the raw data of the image.
 			</description>
 		</method>
 		<method name="get_format" qualifiers="const">
 			<return type="int" enum="Image.Format">
 			</return>
 			<description>
-				Return the format of the [code]Image[/code], one of [code]Image[/code].FORMAT_*.
+				Returns the format of the image.
```

Comment:
- Use possessive over "of the"

Returns the image's format. See [code]FORMAT_*[/code] constants.